charts/vector/README.md

```diff
@@ -1,6 +1,6 @@
 # Vector
 
-![Version: 0.1.0-beta.2](https://img.shields.io/badge/Version-0.1.0--beta.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.17.3-distroless-libc](https://img.shields.io/badge/AppVersion-0.17.3--distroless--libc-informational?style=flat-square)
+![Version: 0.1.0-beta.3](https://img.shields.io/badge/Version-0.1.0--beta.3-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.17.3-distroless-libc](https://img.shields.io/badge/AppVersion-0.17.3--distroless--libc-informational?style=flat-square)
```

Comment:
Still wondering if this'll conflict with https://github.com/vectordotdev/helm-charts/pull/104